src/components/Header.css

```diff
@@ -0,0 +1,126 @@
+/*NOTE! 
+The header does not really look like the design. 
+This is due to a desicion me and the designer made
+about changing the typeface to a free one, so I could get
+it from google fonts. This did not oly effect the typeface, 
+but also the size of the hamburger-menu, we decided it would look
+nicer if the height of the menu would match the x-height of the new font.
+*/
```

Comment:
Really good that you guys had a conversation about the design, good communication is key! Tricky to implement a "non-free" font family 🙈